docs/jenkins.md

```diff
@@ -0,0 +1,103 @@
+# Samples for Jenkins pipelines
+
+These are configuration examples for running a self-hosted Renovate on Jenkins pipelines. This example accesses a privately hosted GitLab instance.
+See [self-hosting](https://docs.renovatebot.com/getting-started/running/#self-hosting-renovate) doc for additional information.
+
+## Renovate with mapped Docker socket
+
+The following pipeline runs Renovate normally on the default branch (eg. `main` or `master`).
+
+### Setup
+
+- Fill in the `GIT_AUTHOR_` and `GIT_COMMITTER_` fields with the same account data
+- `RENOVATE_ENDPOINT`: your GitLab API endpoint
+- `RENOVATE_REPOSITORIES`: your repositories to renovate
+- `gitLabConnection`: your GitLab domain
+
+### Variables
+
+- `RENOVATE_TOKEN`: access token for renovate to gitlab api (**required**)
+- `GITHUB_COM_TOKEN`: suppress github api rate limits (**required**)
+- `RENOVATE_EXTRA_FLAGS`: pass additional commandline args (**optional**)
+
+### Jenkinsfile
+```yml
+#!groovy
+
+pipeline {
+    agent {
+        docker {
+            image 'renovate/renovate:32.32.5'
+            args '-v /var/run/docker.sock:/var/run/docker.sock -v /tmp:/tmp --group-add 0'
+        }
+    }
+
+    environment {
+        CONFIGURATION = 'Release'
+        TZ = 'Europe/Berlin'
+        ENV = '/usr/local/etc/env'
+        RENOVATE_TOKEN = credentials('RENOVATE_TOKEN')
+        GITHUB_COM_TOKEN = credentials('GITHUB_COM_TOKEN')
+        LOG_LEVEL = 'debug'
+        GIT_AUTHOR_NAME = 'Renovate Bot'
+        GIT_AUTHOR_EMAIL = 'bot@example.com'
+        GIT_COMMITTER_NAME = 'Renovate Bot'
+        GIT_COMMITTER_EMAIL = 'bot@example.com'
+        RENOVATE_PLATFORM = 'gitlab'
+        RENOVATE_ENDPOINT = 'https://git.example.com/api/v4/'
+        RENOVATE_REPOSITORIES = 'user1/repo1, user2/repo2'
+        RENOVATE_ONBOARDING_CONFIG = '{ "extends":["config:base"] }'
+        RENOVATE_BINARY_SOURCE = 'install'
```

Comment:
you don't need to map docker socket for this mode.

you should add two samples
1. docker mode with slim image
2. install mode with full image

Reply:
@viceice , I'm sorry. I'm not an expert in these things and have only limited time. This is what I did, what I can vouch for in terms of that it works and what I can offer you at this point. Please accept this as a base which can then be extended by someone else.

Reply:
```suggestion
```

Simply remove, because you like to use the docker sidecar